lib-ui-compose/src/commonMain/kotlin/com/ustadmobile/libuicompose/view/report/graphs/Graphs.kt

```diff
@@ -0,0 +1,454 @@
+package com.ustadmobile.libuicompose.view.report.graphs
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.ustadmobile.core.domain.report.model.GraphSeries
+import com.ustadmobile.core.domain.report.model.ReportOptions2
+import com.ustadmobile.core.domain.report.model.ReportXAxis
+import com.ustadmobile.core.domain.report.model.SeriesType
+import com.ustadmobile.core.domain.report.model.YAxisTypes
+import com.ustadmobile.core.util.report.ReportFormatter
+import com.ustadmobile.libuicompose.util.ext.defaultItemPadding
+import com.ustadmobile.libuicompose.view.report.detail.getGenderLabel
+import io.github.koalaplot.core.ChartLayout
+import io.github.koalaplot.core.Symbol
+import io.github.koalaplot.core.bar.DefaultVerticalBar
+import io.github.koalaplot.core.bar.DefaultVerticalBarPlotGroupedPointEntry
+import io.github.koalaplot.core.bar.DefaultVerticalBarPosition
+import io.github.koalaplot.core.bar.GroupedVerticalBarPlot
+import io.github.koalaplot.core.legend.FlowLegend
+import io.github.koalaplot.core.legend.LegendLocation
+import io.github.koalaplot.core.line.LinePlot
+import io.github.koalaplot.core.style.LineStyle
+import io.github.koalaplot.core.util.ExperimentalKoalaPlotApi
+import io.github.koalaplot.core.util.VerticalRotation
+import io.github.koalaplot.core.util.generateHueColorPalette
+import io.github.koalaplot.core.util.rotateVertically
+import io.github.koalaplot.core.xygraph.DefaultPoint
+import io.github.koalaplot.core.xygraph.FloatLinearAxisModel
+import io.github.koalaplot.core.xygraph.XYGraph
+
+private const val HOUR_UNIT = "hr"
```

Comment:
These are hardcoded English strings. This is clearly not meeting coding styles and basic good practice.